buttons.js

```diff
@@ -0,0 +1,162 @@
+//     return (tv.originalStock - tv.sold) === 0;
+//   })
+//   console.log('\nSold out tv\'s: ');
+//   console.log(soldOutTvs);
+// }
+//
+//
+//
+// AMBILIGHT TV'S
+//
+//
+// const buttonOnlyAmbiTvs = document.getElementById("show-only-ambi-tvs")
+//
+// buttonOnlyAmbiTvs.addEventListener("click", calcOnlyAmbiTvs);
+//
+// // Kopie van code opdracht 1c > omgezet naar functie
+// function calcOnlyAmbiTvs() {
+//   const onlyAmbiTv = inventory.filter((tv) => {
+//     return tv.options.ambiLight === true;
+//   })
+//   console.log('\nOnly Ambilight tv\'s');
+//   console.log(onlyAmbiTv);
+// }
+//
+//
+// SORTEER OP PRIJS (VAN LAAG NAAR HOOG)
+//
+//
+// const buttonSortTvsLowToHigh = document.getElementById("show-tvs-sorted-low-to-high")
+//
+// buttonSortTvsLowToHigh.addEventListener("click", calcSortTvsLowToHigh);
+//
+// // Kopie van code opdracht 1d > omgezet naar functie
+// function calcSortTvsLowToHigh() {
+//   const priceLowToHigh = inventory.sort((a, b) => a.price - b.price)
+//
+//   console.log('\nPrice of tv\'s from low to high: ');
+//   console.log(priceLowToHigh);
+// }
+// =============  CODE UITWERKING >  UITGECOMMENT IVM DUBBELINGEN  =============
+
+
+
+// OPDRACHT 2.
+// Zorg er nu voor, in plaats van dat de uitkomsten in de console worden gelogd, dat de uitkomsten worden meegegeven aan
+// jouw functie uit 4e zodat de resultaten daadwerkelijk op de pagina weergegeven worden!
+
+// Maak een flexbox van de button container met lichtblauwe achtergrond, 20 px padding vanaf de inhoud
+// Maak een flexbox van de buttons zelf, achtergrondkleur donkerder blauw, met ruimte voor antwoordregel onder de button
+// Maak antwoordregel onder de buttons. Antwoord: ___ > Antwoord zelf aparte tag en in andere kleur
+// Link JS code aan antwoord tag
+
+
+// UITVERKOCHTE EXEMPLAREN
+
+// Button link en gedrag
+const buttonSoldOutTvs = document.getElementById("show-soldout-tvs")
+buttonSoldOutTvs.addEventListener("click", calcSoldOutTvs);
+
+// Antwoordregel die aangepast gaat worden na button click
+let answerSoldOutTvs = document.getElementById("answer-sold-tvs");
+
+// Berekent welke tv's zijn uitverkocht en voert functies uit die het antwoord tonen in de browser
+function calcSoldOutTvs() {
+  const soldOutTvs = inventory.filter((tv) => {
+    return (tv.originalStock - tv.sold) === 0;
+  })
+  const answer = returnNameAndBrandOfObject(soldOutTvs);
+  changeAnswer(answerSoldOutTvs, answer);
+}
+
+
+
+// AMBILIGHT TV'S
+
+// Button link en gedrag
+const buttonOnlyAmbiTvs = document.getElementById("show-only-ambi-tvs")
+buttonOnlyAmbiTvs.addEventListener("click", calcOnlyAmbiTvs);
+
+// Antwoordregel die aangepast gaat worden na button click
+let answerAmbiTvs = document.getElementById("answer-ambi-tvs");
+
+
+// Selecteert tv's die AmbiLight hebben en voert functies uit die het antwoord tonen in de browser
+function calcOnlyAmbiTvs() {
+  const onlyAmbiTv = inventory.filter((tv) => {
+    return tv.options.ambiLight === true;
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return tv.options.ambiLight;
```